Dockerfile

```diff
@@ -1,70 +1,68 @@
-FROM python:3.4
+FROM ubuntu:16.04
 MAINTAINER Benjamin Hutchins <ben@hutchins.co>
 
-# Install nginx
-ENV NGINX_VERSION 1.9.7-1~jessie
+ENV LANG de_DE.UTF-8
```

Comment:
Not sure if the default should be German... I mean, I like German and all, but not sure if it's popular enough of a language to be the default.